fujiji-client/src/utils/fujijiConfig.js

```diff
@@ -52,4 +57,4 @@ export const provinces = [
   },
 ];
 
-export const conditions = ['New', 'Used', 'Refurbished'];
+export const conditions = ['Select', 'New', 'Used', 'Refurbished'];
```

Comment:
What does `Select` condition mean?

Reply:
We actually need that for future changes to have key value pairs for empty or no choice. I will create a issue explaining this.